src/components/SearchFilter/SearchFilter.tsx

```diff
@@ -0,0 +1,229 @@
+              ...filterItems,
+              application: !filterItems.application,
+            })
+            updateCheckedItems(`category:${e.target.value}`)
+          }}
+        />
+        <Checkbox
+          id="news-filter"
+          name="news"
+          label="News"
+          value="news"
+          className={styles.checkbox}
+          checked={filterItems.news}
+          onChange={(e) => {
+            setFilterItems({
+              ...filterItems,
+              news: !filterItems.news,
+            })
+            updateCheckedItems(`category:${e.target.value}`)
+          }}
+        />
+        <Checkbox
+          id="documentation-filter"
+          name="documentation"
+          label="Documentation"
+          value="documentation"
+          className={styles.checkbox}
+          checked={filterItems.documentation}
+          onChange={(e) => {
+            setFilterItems({
+              ...filterItems,
+              documentation: !filterItems.documentation,
+            })
+            updateCheckedItems(`category:${e.target.value}`)
+          }}
+        />
+      </Grid>
+      <Grid col="auto">
+        <Label htmlFor="label-dropdown" className={styles.subHeader}>
+          Labels:
+        </Label>
+        <Dropdown
+          id="label-dropdown"
+          name="label-dropdown"
+          data-testid="label-dropdown"
+          value={filterItems.dropdown ? filterItems.dropdown : 'default'}
+          onChange={(e) => {
+            // Need to remove any previous label and add the newly selected one
+            const labelToAdd = e.target.value
+
+            // Filter out any previous label from the query
+            const filteredArray = searchPageFilters.filter(
+              (label) => !label.includes('label:')
+            )
+
+            // Add the new label to the array
+            filteredArray.push(`label:${labelToAdd}`)
+            setSearchPageFilters(filteredArray)
+
+            // Update the state of the dropdown
+            setFilterItems({
+              ...filterItems,
+              dropdown: labelToAdd,
+            })
+          }}>
+          <option value="default" disabled>
+            None applied
+          </option>
+          {labels.map(({ name }) => (
+            <option
+              key={name}
+              value={name}
+              aria-selected={filterItems.dropdown === name ? true : false}>
+              {name}
+            </option>
+          ))}
+        </Dropdown>
+      </Grid>
+
+      <form
+        role="search"
+        method="get"
+        action="/search"
+        onSubmit={(e) => {
+          // Before submitting the form, we need to add the current value of searchQuery to the
+          // end of the searchPageFilters array
+          setSearchPageFilters([...searchPageFilters, searchQuery])
+
+          // Save the filterItems state to local storage
+          localStorage.setItem('filterItems', JSON.stringify(filterItems))
```

Comment:
This is so that when the search is submitted and the page refreshes, the user's filter selections are still visible in the UI. I thought it might be confusing for the user if the items they selected in the filter reset after the search, potentially causing them to think that the filter was not applied.